modules/exploits/linux/local/runc_cwd_priv_esc.rb

```diff
@@ -57,36 +58,34 @@ def initialize(info = {})
       )
     )
     register_advanced_options [
-      OptString.new('WritableDir', [ true, 'A directory where we can write and execute files', '/tmp' ]),
+      OptString.new('WRITABLEDIR', [ true, 'A directory where we can write and execute files', '/tmp' ]),
```

Comment:
Is there a reason to change `WritableDir` to uppercase? `WritableDir` is standard across local exploitation modules.

If this option is changed it should also be changed in the associated module documentation.

Reply:
This is again due to inexperience. If it is a standard then it would make no sense for me to change it. Reverted.